espnet/asr/pytorch_backend/asr.py

```diff
@@ -957,6 +957,15 @@ def recog(args):
     model, train_args = load_trained_model(args.model, training=False)
     assert isinstance(model, ASRInterface)
     model.recog_args = args
+    if args.quantize_asr_model:
+        assert (
+            "transducer" not in args.model_module
+        ), "Quantization of transducer model is not supported yet."
+        logging.info("Use quantized asr model for decoding")
+        dtype = getattr(torch, args.quantize_dtype)
+        model = torch.quantization.quantize_dynamic(
+            model, args.quantize_config, dtype=dtype
+        )
```

Comment:
```suggestion
    if args.quantize_asr_model:
        assert (
            "transducer" not in train_args.model_module
        ), "Quantization of transducer model is not supported yet."
        logging.info("Use quantized asr model for decoding")

        dtype = getattr(torch, args.quantize_dtype)
        
        if args.quantize_config is not None:                                                                                   
            q_config = set([getattr(torch.nn, q) for q in args.quantize_config])                                               
        else:                                                                                                                  
            q_config = {torch.nn.Linear}                                                                                       
                                                                                                                               
        model = torch.quantization.quantize_dynamic(                                                                           
            model, q_config, dtype=dtype                                                                                       
        )
```

Reply:
Sorry, I forgot `quantize-config` parameter was used in other case. Thus:

```suggestion

    if args.quantize_config is not None:                                                                                   
        q_config = set([getattr(torch.nn, q) for q in args.quantize_config])                                               
    else:                                                                                                                  
        q_config = {torch.nn.Linear} 

    if args.quantize_asr_model:
        assert (
            "transducer" not in train_args.model_module
        ), "Quantization of transducer model is not supported yet."
        logging.info("Use quantized asr model for decoding")

        dtype = getattr(torch, args.quantize_dtype)                                      
        model = torch.quantization.quantize_dynamic(                                                                           
            model, q_config, dtype=dtype                                                                                       
        )
```